### PR TITLE
refactor(yaml): 🔍  we want live security updates

### PR DESCRIPTION
**Pull request by mitchelvanbever**

### CODE REVIEW

**Comment by mitchelvanbever** [comment]
This should do the trick. We couild also add default labels, branch name and/or reviewers if prefered.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 version: 2
 updates:
-  - package-ecosystem: "npm"
+  - package-ecosystem: "yarn"
```
Comment (ggurkal) [inline .github/dependabot.yml:3]: i tried `yarn` before, but apparently it's not valid. should be set back to `npm`.

**Review by ggurkal — CHANGES_REQUESTED** [review]

**Comment by ggurkal** [comment]
The config becomes a mix of v1 and v2, so we will get errors. Validation can be done here: https://dependabot.com/docs/config-file/validator/

I'm a bit confused about dependabot as those two versions work quite different than each other. The changes are for v1 but I don't know if it's possible to use v1 with github as is. 🤔

**Comment by ggurkal** [comment]
Seems like github supports only v2, which I believe doesn't create PRs only security updates. At least I couldn't find it...

**Comment by ggurkal** [comment]
@mitchelvanbever seems like v1 is not supported anymore. so we have to revert the changes, but we can make some additions, like settings `beta` as the branch.

we can also try settings `versioning-strategy: increase-if-necessary` to see if it works as _security updates_.

**Comment by mitchelvanbever** [comment]
> @mitchelvanbever seems like v1 is not supported anymore. so we have to revert the changes, but we can make some additions, like settings `beta` as the branch.
> 
> we can also try settings `versioning-strategy: increase-if-necessary` to see if it works as _security updates_.

Okay let's revert the changes and try that approach. I did also enable the dependebot-preview thingy for all repo's that are not managed through vercel, so maybe we don't even need the yml.

**Pull request merged**